docs/integrations/sources/freshbooks.md

```diff
@@ -0,0 +1,40 @@
+# FreshBooks
+FreshBooks connector  seamlessly syncs invoicing, expenses, and client data from FreshBooks into data warehouses or analytics platforms. 
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `client_id` | `string` | Client ID.  |  |
+| `client_secret` | `string` | Client secret.  |  |
+| `redirect_uri` | `string` | Redirect Uri.  |  |
```

Comment:
Provide more information about Redirect URI, it need to be a variable?

Reply:
@marcosmarxm yes as it states here in the pic
<img width="1335" alt="Screenshot 2024-11-07 at 9 25 22 AM" src="https://github.com/user-attachments/assets/f6b3e57c-5340-4a46-bfe2-da6744bc8dfd">